elytron/src/main/java/org/wildfly/extension/elytron/DistributedRealmDefinition.java

```diff
@@ -65,7 +71,19 @@
             .setRestartAllServices()
             .build();
 
-    static final AttributeDefinition[] ATTRIBUTES = new AttributeDefinition[] {REALMS};
+    static final SimpleAttributeDefinition IGNORE_UNAVAILABLE_REALMS = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.IGNORE_UNAVAILABLE_REALMS, ModelType.BOOLEAN, true)
+            .setDefaultValue(ModelNode.FALSE)
+            .setAllowExpression(true)
```

Comment:
Because this new attribute allows expressions, we'll also want to update https://github.com/wildfly/wildfly-core/blob/main/elytron/src/test/resources/org/wildfly/extension/elytron/elytron-expressions.xml to add a `distributed-realm` that uses an expression for this attribute.